src/components/Camera/index.js

```diff
@@ -46,6 +47,7 @@ const CameraPure = ({
           {...{onUserMedia, ref: webcamRef, onFailure}}
         />
       </div>
+      <div id='cameraViewAriaLabel' aria-label={translate('accessibility.camera_view')}></div>
```

Comment:
Why do we need this id?

Reply:
Only because the area it is describing is actually the `webcamContainer` above it. When the ARIA label was part of that `div` itself the Android screen readers weren't picking it up even with an accompanying ARIA role.